app/mailers/notify_pq_mailer.rb

```diff
@@ -20,7 +20,7 @@ def acceptance_email(pq:, action_officer:, email:)
       answer_by: pq.minister&.name || "",
       internal_deadline: internal_deadline_text(pq) || "",
       date_to_parliament: date_to_parliament_text(pq) || "",
-      cc_list: cc_list(pq, action_officer) || "",
+      cc_list: action_officer.deputy_director.email || "",
```

Comment:
In the cc_list method, it handled the case where the action officer did not have a deputy director assigned.